src/AppBundle/Resources/views/Admin/ReportSubmission/index.html.twig

```diff
@@ -146,7 +146,9 @@
                             </td>
                         {% endif %}
                         <td class="width-tenth change-answer">
-                            <a href="{{ path('admin_documents_download', {'reportSubmissionId': reportSubmission.id}) }}" class="behat-link-download">Download</a><br>
+                            {% if reportSubmission.downloadable %}
```

Comment:
isDownloadable?

Reply:
twig auto-finds the method getDownloadable or isDownloadable